tools/esdlcmd/esdlcmd_shell.cpp

```diff
@@ -180,7 +180,7 @@ bool EsdlCMDShell::parseCommandLineOptions(ArgvIterator &iter)
 void EsdlCMDShell::usage()
 {
     fprintf(stdout,"\nUsage:\n"
-        "    esdl [--version] <command> [<args>]\n\n"
```

Comment:
We shouldn't remove the esdl --version command (this prints out the version of the tool)